desktop_files/package.json

```diff
@@ -55,7 +55,7 @@
     "querystring-es3": "0.2.1",
     "react": "16.4.1",
     "react-dom": "16.4.2",
-    "react-native": "git+https://github.com/status-im/react-native-desktop.git#master",
+    "react-native": "git+https://github.com/status-im/react-native-desktop.git#420798a31ea89f8d2de007f7321b6db64a444685",
```

Comment:
Hi!
What branch it points to?
It should be still `master` I believe.

Reply:
@vkjr hi! it points to latest commit in master https://github.com/status-im/react-native-desktop/commit/420798a31ea89f8d2de007f7321b6db64a444685
I changed it to be refererring exact commit because reference to master always change, but reference to exact commit is more explicit way to say what version you're using.

Reply:
@vkjr makes sense to use descriptive tags. For example, we could use git tags to specify version like `v0.56.1`.

@MaxRis @siphiuel what do you guys think about using git tags for this?

Reply:
Good idea.

Reply:
Changed to git tag `v0.56.1`  https://github.com/status-im/react-native-desktop/tree/v0.56.1

Reply:
Thanks for changing!
Just to note, `0.56.1` is a version of original `react-native`we are basing on. But within this version we have a lot of commits in `react-native-desktop`, so next tags will be more granular, like `v0.56.1-1`, `v0.56.1-2`, etc..